### DISCUSSION TITLE
Location of IP Address

### DISCUSSION THREAD

**Discussion by jlucas090866** [nathanmarlor/foxess_modbus#333] [Support]
Hi - please can you tell me where the IP address of the inverter is stored.

**Comment by canton7** [comment]
Can you be more specific?

Are you trying to find out by looking at the inverter what ip it's been given? Are you asking where this integration stores the ip address that you input when you set up the integration? Something else?

**Comment by jlucas090866** [comment]
Are you asking where this integration stores the ip address that you input when you set up the integration? YES

I’ve changed some network settings and this need to update the integration without having to reconfigure it.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by canton7** [reply]
See https://github.com/nathanmarlor/foxess_modbus/wiki/FAQs#how-do-i-change-my-inverter-settings-
